docs/release_notes.md

```diff
@@ -1,5 +1,160 @@
 #
 
+## 5.0.0
+
+Release Date: 2025-05-XX
+
+The iRODS Consortium and RENCI are pleased to announce iRODS 5.0.0.
+
+This release represents a significant effort to make iRODS what users expect from an enterprise grade data management platform. Great focus has been put into making iRODS 5 easy to deploy and manage in a variety of environments.
+
+Notable updates include:
+
+- **Revamped Startup/Shutdown Process** - The startup/shutdown process has been fully redesigned. The server honors signals in the expected manner and also communicates its status to the service manager. Administrators will find that iRODS 5 is significantly easier to deploy in the cloud, containers, you name it.
+- **Unified Server Configuration** - All server configuration options, excluding zone-wide options, have been consolidated into `server_config.json`. The server no longer relies on the service account's `irods_environment.json` file.
+- **OpenSSL 3** - The server has been migrated from OpenSSL 1 for improved security and compatibility.
+- **Access Time Tracking** - The server has been updated to track access time for all replicas. Updates to the access time are applied in the background to maintain performance. Access time information can be viewed using GenQuery.
+- **Build against libstdc++** - Building against libstdc++ reduces the need to build so many externals packages. As a result, some packages have been migrated to system-provided packages.
+- **GenQuery1** - A brand new flex-bison parser has been added for GenQuery1. This new parser resolves all long-standing bug reports involving embedded single quotes, special characters, reserved keywords, etc.
+- **GenQuery2** - Users have been granted full control over the placement of the `DISTINCT` keyword. This is a major departure from GenQuery1. Unlink GenQuery1, GenQuery2 makes queries explicit and easier to reason about.
```

Comment:
- Highlight difference in how the DISTINCT keyword is handled as compared to GenQuery1
- Highlight backward compatibility breakage due to the DISTINCT keyword

Reply:
Done.